README.md

```diff
@@ -47,6 +47,7 @@ Prerequisites:
    https://docs.docker.com/engine/installation/linux/ubuntu/#install-using-the-repository)
 
    * Also add current user to 'docker' group (use `sudo usermod -a -G docker $USER` and then relogin)
+3. Install Mozilla WebDriver: `sudo apt install firefox-geckodriver`
```

Comment:
Don't forget to fix the 'universum_log_collapser' section further in Readme file

Reply:
I've planned to remove all related to Jenkins plugin in one PR when HTML log functionality will be ready.
Do you want some update or removal right now?

Reply:
No, separate PR is a good idea, thanks.